src/base/search/searchpluginmanager.cpp

```diff
@@ -193,11 +199,12 @@ void SearchPluginManager::installPlugin(const QString &source)
 void SearchPluginManager::installPlugin_impl(const QString &name, const QString &path)
 {
     PluginVersion newVersion = getPluginVersion(path);
-    qDebug() << "Version to be installed:" << newVersion;
+    Logger *const logger = Logger::instance();
+    logger->addMessage(tr("Updating plugin %1 to version %2").arg(name).arg(newVersion), Log::INFO);
 
     PluginInfo *plugin = pluginInfo(name);
     if (plugin && !(plugin->version < newVersion)) {
-        qDebug("Apparently update is not needed, we have a more recent version");
+        logger->addMessage(tr("Plugin already at version %1, which is greater than %2").arg(plugin->version).arg(newVersion), Log::INFO);
```

Comment:
Many of these log messages are just a trash. I would leave only really necessary ones.

Reply:
These messages are valuable for knowing what happens after triggering an action. But maybe the log isn't the right place for them. I think I might have to handle this in the webui...